.github/workflows/connectors_version_increment.yml

```diff
@@ -0,0 +1,71 @@
+name: Connectors Version Bump
+
+concurrency:
+  # This is the name of the concurrency group. It is used to prevent concurrent runs of the same workflow.
+  #
+  # - github.head_ref is only defined on PR runs, it makes sure that the concurrency group is unique for pull requests
+  #  ensuring that only one run per pull request is active at a time.
+  #
+  # - github.run_id is defined on all runs, it makes sure that the concurrency group is unique for workflow dispatches.
+  # This allows us to run multiple workflow dispatches in parallel.
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      connectors: ${{ steps.changes.outputs.connectors }}
+    permissions:
+      statuses: write
+    steps:
+      - name: Checkout Airbyte
+        if: github.event_name != 'pull_request'
+        uses: actions/checkout@v3
```

Comment:
I don't get why we'd need to checkout. And I don't think the condition will ever be met as the `on` parameters are only set to `pull_request`